### DISCUSSION TITLE
[Kavita] Hide chapters you've already read

### DISCUSSION THREAD

**Discussion by tomerh2001** [Kareadita/Kavita#2991] [Ideas]
### Idea Description

It would be nice if the UI automatically won't display chapters I've already read, or at least give me the option to hide them

### Idea Category

Web UI

### Duration of Using Kavita

6 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by wcamicase** [comment]
Came here to suggest this, glad I'm not the only one!
I'd even go a step further and say that we should have a couple of filters in general: Show only unread chapters, show only read chapters and maybe even be able to order chapters in an inverted way, from finish to start, per se.

It's a needed functionality, and I want to show my support here.
